index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Project1</title>
+		<script src="https://kit.fontawesome.com/62fc5eef3c.js" crossorigin="anonymous"></script>
+	</head>
+	<body>
+		<!--Navbar -->
```

Comment:
i appreciate the intention of adding this type of comment, however it is not usually done, because the code should be easy enough to navigate without comments like this one, so you should remove it